app/lib/three_scale/analytics/user_tracking.rb

```diff
@@ -102,7 +102,7 @@ def basic_traits
 
             account_id: @account.id,
             domain: @account.internal_domain,
-            self_domain: @account.internal_self_domain
+            self_domain: @account.provider? ? @account.internal_admin_domain : nil
```

Comment:
Wanted to double check there was a test or something for this.

Reply:
I think we had tests around this, that's how we noticed I guess. @josemigallas , do you remember?